manifests/h/Heizi/HeiziFlashTools/0.0.8/Heizi.HeiziFlashTools.locale.zh-CN.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Heizi.HeiziFlashTools
+PackageVersion: 0.0.8
+PackageLocale: zh-CN
+Publisher: Heizi
+PackageName: HeiziFlashTools
+License: ' GPL-3.0 License'

```

```suggestion
License: GPL-3.0
LicenseUrl: https://github.com/ElisaMin/Heizi-Flashing-Tools/blob/main/LICENSE
```
* Quotes are not needed
* Added License Url